packages/esm-ohri-pmtct-app/src/views/mch-summary/mch-summary.component.tsx

```diff
@@ -5,14 +5,29 @@ import styles from '../common.scss';
 import { PatientChartProps } from '@ohri/openmrs-esm-ohri-commons-lib';
 import CurrentPregnancy from './tabs/current-pregnancy.component';
 import HivExposedInfant from './tabs/hiv-exposed-infant.component';
-import { usePatient } from '@openmrs/esm-framework';
+import { usePatient, useConfig } from '@openmrs/esm-framework';
 import dayjs from 'dayjs';
 
 const MaternalSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
+  const { identifiersTypes } = useConfig();
   const { patient, isLoading } = usePatient(patientUuid);
   const dob = patient?.birthDate;
   const age = useMemo(() => dayjs().diff(dayjs(patient?.birthDate), 'year'), [patient?.birthDate]);
+  const [pTrackerId, setPtrackerId] = useState('');
+
+  useEffect(() => {
+    if (patient) {
+      const lastIdentifier = patient.identifier
```

Comment:
I have some reservations on this, the last pTracker id will probably have the latest date and one that is no longer valid has an end date., unless a patient can have more than one pTrackerId.

You can have a look at this file, they are fetching identifiers based on types : https://github.com/openmrs/openmrs-react-components/blob/0a739855da0262b6cb79f7e27ae5ff62932232f4/src/domain/patient/patientUtil.js#L108

`patient.identifiers.find((identifier) => identifier.type.id === identifiersTypes.PTrackerIdentifierType);`
should work